brainfuck/executor_test.go

```diff
@@ -0,0 +1,56 @@
+package brainfuck
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"testing"
+)
+
+func captureStdout(f func()) string {
+	old := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	f()
+
+	w.Close()
+	os.Stdout = old
+
+	var buf bytes.Buffer
+	io.Copy(&buf, r)
```

Comment:
`io.Copy` returns `written int64, err error`. Pls don't ignore that.

Reply:
Fixed